server/middlewares/ArticleValidation.js

```diff
@@ -184,6 +184,50 @@ class ArticleValidation {
     ).optional({ checkFalsy: false }).isAlpha();
   }
 
+  /**
+   * This method validates the comment id
+   * @param {object} req - The request object
+   * @returns {void}
+   */
+  static validateCommentId(req) {
+    req.checkParams(
+      'commentId',
+      'commentId must be an integer'
+    ).isInt();
+    req.checkParams(
+      'commentId',
+      'commentId must be greater than 0'
+    ).isInt({ gt: 0 });
+  }
+
+  /**
+   * This method validates the articleId
+   * @param {object} req - The request object
+   * @returns {void}
+   */
+  static validateArticleId(req) {
+    req.checkParams(
+      'articleId',
+      'articleId must be an integer'
+    ).isInt();
+    req.checkParams(
+      'articleId',
+      'articleId must be greater than 0'
+    ).isInt({ gt: 0 });
+  }
```

Comment:
These two methods here are doing what the `validateResourceId` middleware already does. You already have that on your route. I don't think there's any need to have this as well.

Reply:
True removed.